wsi/drm/drmdisplay.cpp

```diff
@@ -135,11 +135,13 @@ bool DrmDisplay::ConnectDisplay(const drmModeModeInfo &mode_info,
   SetPowerMode(power_mode_);
 
   // This is a valid case on DSI panels.
-  if (!broadcastrgb_props)
+  if (broadcastrgb_props == NULL)
```

Comment:
Print error message here so that we're aware of problems during runtime.

Reply:
Based on the comment above, I have to assume that this isn't an error path. Maybe an info print would be suitable.